blocks/footer/footer.css

```
@@ -16,5 +17,134 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer a {
+  color: #fff;
+}
+
+footer li a:any-link {
+  color: white;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
```

let's move the left and right-side styles up here, so that parents are defined before the children. 
I also removed flex where it can be done a bit simpler. 

```suggestion


.footer-container .left-side {
  align-self: flex-start;
}

.footer-container .right-side {
  text-align: right;
  margin: 0 20px;
}

.footer-container .right-side img {
  width: 200px;
  height: auto;
}
```